services/snyk/snyk-vulnerability-github.service.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+const SynkVulnerabilityBase = require('./snyk-vulnerability-base')
+
+module.exports = class SnykVulnerabilityGitHub extends SynkVulnerabilityBase {
+  static get route() {
+    return {
+      base: 'snyk/vulnerabilities/github',
+      format: '([^/]+)/([^/]+)(?:/(.+))?',
```

Comment:
I ended up putting the GitHub and npm package badges in different files, almost entirely because their parameters are so different. If there's a way of supporting both in the same Service definition I'm happy to change it. 

I also included `vulnerability` in the file & service name as Snyk has other data (like licenses) that could potentially be added in the future